### PR TITLE
📝 Document variables for static spaces

### PR DESCRIPTION
**Pull request by coyotte508**
Follow up to https://github.com/huggingface/moon-landing/pull/8444

### CODE REVIEW

File: docs/hub/spaces-sdks-static.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ Here are some examples of Spaces using custom HTML:
 
 * [Smarter NPC](https://huggingface.co/spaces/mishig/smarter_npc): Display a PlayCanvas project with an iframe in Spaces.
 * [Huggingfab](https://huggingface.co/spaces/pierreant-p/huggingfab): Display a Sketchfab model in Spaces.
+
+## Space variables
+
+Custom [variables](./spaces-overview#managing-secrets) are available in your Space. OAuth information such as the client ID and scope are also available as environment variables, if you have [enabled OAuth](./spaces-oauth) for your Space.
+
+To use these variables in JavaScript, you can use the `window.huggingface.variables` object. For example, to access the `OAUTH_CLIENT_ID` variable, you can use `window.huggingface.variables.OAUTH_CLIENT_ID`.
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-static.md:14]: Maybe share an example space for reference code

**Review by osanseviero — APPROVED** [review]
:+1: :fire:

File: docs/hub/spaces-sdks-static.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ Here are some examples of Spaces using custom HTML:
 
 * [Smarter NPC](https://huggingface.co/spaces/mishig/smarter_npc): Display a PlayCanvas project with an iframe in Spaces.
 * [Huggingfab](https://huggingface.co/spaces/pierreant-p/huggingfab): Display a Sketchfab model in Spaces.
+
+## Space variables
+
+Custom [variables](./spaces-overview#managing-secrets) are available in your Space. OAuth information such as the client ID and scope are also available as environment variables, if you have [enabled OAuth](./spaces-oauth) for your Space.
```
Comment (SBrandeis) [inline docs/hub/spaces-sdks-static.md:12]: Maybe clearer this way? (not sure)
```suggestion
Custom [environment variables](./spaces-overview#managing-secrets) can be passed to your Space. OAuth information such as the client ID and scope are also available as environment variables, if you have [enabled OAuth](./spaces-oauth) for your Space.
```

File: docs/hub/spaces-overview.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ If your app requires environment variables (for instance, secret keys or tokens)
 </div>
 
 
-Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables. For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
+Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables. For Static Spaces, they are available through client-side JavaScript in `window.huggingface.variables`. For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
```
Comment (SBrandeis) [inline docs/hub/spaces-overview.md:69]: (nit) jumping lines to spearate more Docker vs Static

A link to `spaces-sdk-static` could be nice too, wdyt?

```suggestion
Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables.

For Static Spaces, they are available through client-side JavaScript in `window.huggingface.variables`.

For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
```

**Review by SBrandeis — APPROVED** [review]
Some wording suggestions, otherwose LGTM
Thank you!

File: docs/hub/spaces-sdks-static.md [hunk 4]
```diff
@@ -11,4 +11,8 @@ Here are some examples of Spaces using custom HTML:
 
 Custom [environment variables](./spaces-overview#managing-secrets) can be passed to your Space. OAuth information such as the client ID and scope are also available as environment variables, if you have [enabled OAuth](./spaces-oauth) for your Space.
 
-To use these variables in JavaScript, you can use the `window.huggingface.variables` object. For example, to access the `OAUTH_CLIENT_ID` variable, you can use `window.huggingface.variables.OAUTH_CLIENT_ID`.
\ No newline at end of file
+To use these variables in JavaScript, you can use the `window.huggingface.variables` object. For example, to access the `OAUTH_CLIENT_ID` variable, you can use `window.huggingface.variables.OAUTH_CLIENT_ID`.
+
+Here is an example of a Space using custom environment variables and oauth enabled and displaying the variables in the HTML:
+
+* [Static Variables](https://huggingface.co/spaces/coyotte508/static-variables)
```
Comment (SBrandeis) [inline docs/hub/spaces-sdks-static.md:18]: Nice ❤️ 

Let's moev this Space to `huggingface-projects`?

https://huggingface.co/huggingface-projects

**Review by SBrandeis — APPROVED** [review]

File: docs/hub/spaces-sdks-static.md (hunk 4 above)
Comment (julien-c) [reply docs/hub/spaces-sdks-static.md]: was fine to keep on personal namespace too imo

**Pull request merged**